Competition2017/src/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -148,7 +163,6 @@ public void tankDrivePowerMode(double leftPower, double rightPower) {
      * @param motor used to access the motor to set and give data
      */
     private void updateMotorConfig(XCANTalon motor) {
-        motor.configEncoderCodesPerRev((int)encoderCodesProperty.get());
```

Comment:
Why was this removed? Is the idea to do all the conversions ourselves? If so, the unused property should be removed. Also, the value should be explicitly set to 1 so that we aren't using whatever value was last set.